spec/config.yaml

```diff
@@ -632,3 +642,193 @@ mappings:
       objtype: {type: keyword}
       perm: {type: keyword}
       permusers: {type: keyword}
+
+  "sample_set_1":
```

Comment:
You should probably add some `copy_to: agg_fields` on some fields. We use that "agg_fields" field (aggregation of fields) for generic text searches, allowing users to generically text search across many fields

Reply:
wasn't sure if they should end up in the agg_fields, created a separate field for sample specific search